.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.0.0",
-  "packages/api-explorer": "0.9.27",
+  ".": "2.1.0",
+  "packages/api-explorer": "0.9.28",
   "packages/code-editor": "0.1.18",
-  "packages/extension-api-explorer": "22.0.0",
-  "packages/extension-sdk": "22.0.0",
-  "packages/extension-sdk-react": "22.0.0",
-  "packages/extension-utils": "0.1.4",
-  "packages/hackathon": "22.0.0",
-  "packages/run-it": "0.9.27",
-  "packages/sdk": "22.0.0",
+  "packages/extension-api-explorer": "22.2.0",
+  "packages/extension-sdk": "22.2.0",
+  "packages/extension-sdk-react": "22.2.0",
+  "packages/extension-utils": "0.1.5",
+  "packages/hackathon": "22.2.0",
+  "packages/run-it": "0.9.28",
+  "packages/sdk": "22.2.0",
   "packages/sdk-codegen": "21.4.0",
-  "packages/sdk-codegen-scripts": "21.2.2",
+  "packages/sdk-codegen-scripts": "21.2.3",
```

Comment:
this is correct. `21.2.3` is an odd version but it's this way because we *used* to have it track the Looker calver but now it just semver bumps

Reply:
LGTM